application/org.openjdk.jmc.ide.launch/src/main/java/org/openjdk/jmc/ide/launch/JfrLaunchPage.java

```diff
@@ -119,6 +119,22 @@ public void widgetSelected(SelectionEvent e) {
 		autoOpenCheckbox.setLayoutData(gd);
 	}
 
+	private void createLessThenOracleJdk11(Composite parent, int cols) {
+		lessThanOracleJDK11Checkbox = new Button(parent, SWT.CHECK);
+		lessThanOracleJDK11Checkbox.setText("Oracle JDK < 11");
+		lessThanOracleJDK11Checkbox
+				.setToolTipText("Only check this if it really is an Oracle JDK. For OpenJDK, leave this unchecked.");
```

Comment:
Shouldn't this be in a message bundle ?

Reply:
@aptmac yes, you have described a bug I've forgot to report, I've found it too! It wanted to write a ticket for it. !